bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -82,7 +89,7 @@
  * <p>The exceptions resulting from synchronous calls and error code resulting from
  * asynchronous calls can be found in the class {@link BKException}.
  */
-public class BookKeeper implements AutoCloseable {
+public class BookKeeper implements AutoCloseable, org.apache.bookkeeper.client.api.BookKeeper {
```

Comment:
also, Bookkeeper interface extends AutoCloseable, so its not required to mention it again

Reply:
done